src/routes/RequestsRoute.js

```diff
@@ -1318,13 +1340,41 @@ class RequestsRoute extends React.Component {
     const servicePoints = get(resources, 'servicePoints.records', []);
     const cancellationReasons = get(resources, 'cancellationReasons.records', []);
     const requestCount = get(resources, 'records.other.totalRecords', 0);
+    const isViewPrintDetailsEnabled =
+      get(resources, 'circulationSettings.records[0].value.enablePrintLog') === 'true';
     const initialValues = dupRequest ||
     {
       requestType: DEFAULT_REQUEST_TYPE_VALUE,
       fulfillmentPreference: fulfillmentTypeMap.HOLD_SHELF,
       createTitleLevelRequest: createTitleLevelRequestsByDefault,
     };
 
+    const columnLabels = {
```

Comment:
This long column label config could be moved to a separate file and used here as a function with this argument getIsAllRowsSelected()

Reply:
Hey @SergeYvas , this can be addressed in a separate tech-debt ticket as changing in this PR can affect the other columns + require some efforts.